### ISSUE TITLE
Support MUI v6

### ISSUE THREAD

**Issue by epicstar** [Learus/react-material-ui-carousel#235]
[MUI v6](https://mui.com/blog/material-ui-v6-is-out/) just got released on 2024-08-24. It would be great if we could update the library to support the new version of MUI.

**Comment by epicstar** [comment]
For our use-case the library still works despite the warnings. But our use of the library is very light.

**Comment by agrandin0** [comment]
It would be great to have a compatibility with Mui V6 !

**Comment by GiambaFr** [comment]
I agree, a full support of MIU v6 will be great. Using emotion and removing the MUI Legacy style or styled to have it working in SSR.
